<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>3DUSE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="hsplitter_bottom" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>85</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetTemporal" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBoxTemporalTools">
            <property name="toolTip">
             <string>Toggle temporal tools</string>
            </property>
            <property name="text">
             <string>Enable time</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="toolTip">
             <string>Current date</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="toolTip">
             <string>Play</string>
            </property>
            <property name="text">
             <string>P</string>
            </property>
            <property name="icon">
             <iconset theme="media-playback-start">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Time slider</string>
            </property>
            <property name="maximum">
             <number>109574</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="hsplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="filterLineEdit">
            <property name="toolTip">
             <string>Enter a string for filtering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="toolTip">
             <string>Search nodes with string filtering</string>
            </property>
            <property name="text">
             <string>filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Tree</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QSplitter" name="vsplitter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTreeWidget" name="treeWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="headerDefaultSectionSize">
                <number>200</number>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Type</string>
                </property>
               </column>
              </widget>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_16">
           <attribute name="title">
            <string>Doc</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLogo">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/vcity.qrc">:/images/LirisLogo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainGrid" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <layout class="QGridLayout" name="mainGridLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
    </widget>
    <addaction name="actionLoad"/>
    <addaction name="actionLoad_recursive"/>
    <addaction name="actionLoad_bbox"/>
    <addaction name="menuRecent_files"/>
    <addaction name="separator"/>
    <addaction name="actionExport_citygml"/>
    <addaction name="actionExport_osg"/>
    <addaction name="actionExport_tiled_osga"/>
    <addaction name="actionExport_JSON"/>
    <addaction name="actionExport_OBJ"/>
    <addaction name="actionExport_OBJ_split"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuPicking_mode_2">
     <property name="title">
      <string>Picking mode</string>
     </property>
     <addaction name="actionBuilding"/>
     <addaction name="actionFace"/>
    </widget>
    <addaction name="menuPicking_mode_2"/>
    <addaction name="actionClearSelection"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <widget class="QMenu" name="menuGenerate_LODs">
     <property name="title">
      <string>Generate LODs</string>
     </property>
     <addaction name="actionScene_GenerateLOD0"/>
     <addaction name="actionScene_GenerateLOD1"/>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="menuGenerate_LODs"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuRender">
     <property name="title">
      <string>Render</string>
     </property>
     <addaction name="actionAuto_LOD"/>
     <addaction name="actionForce_LOD0"/>
     <addaction name="actionForce_LOD1"/>
     <addaction name="actionForce_LOD2"/>
     <addaction name="actionForce_LOD3"/>
     <addaction name="actionForce_LOD4"/>
     <addaction name="separator"/>
     <addaction name="actionInfo_bubbles"/>
     <addaction name="actionShadows"/>
     <addaction name="actionShadows_options"/>
    </widget>
    <addaction name="menuRender"/>
    <addaction name="actionShow_temporal_tools"/>
    <addaction name="actionShow_advanced_tools"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFix_building"/>
    <addaction name="actionOBJ_to_CityGML"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Detection"/>
    <addaction name="separator"/>
    <addaction name="actionSplit_CityGML_Buildings"/>
    <addaction name="actionCut_CityGML_with_Shapefile"/>
    <addaction name="actionTiling_CityGML"/>
    <addaction name="actionCut_MNT_with_Shapefile"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Roads_on_MNT"/>
    <addaction name="actionCreate_Vegetation_on_MNT"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionOptim_osg"/>
    <addaction name="actionDump_osg"/>
    <addaction name="actionDump_scene"/>
    <addaction name="actionDump_selected_nodes"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionCompute_files_bbox"/>
    <addaction name="actionTest_1"/>
    <addaction name="actionTest_2"/>
    <addaction name="actionTest_3"/>
    <addaction name="actionTest_4"/>
    <addaction name="actionTest_5"/>
   </widget>
   <widget class="QMenu" name="menuGenerate_2">
    <property name="title">
     <string>Generate</string>
    </property>
    <widget class="QMenu" name="menuLODs">
     <property name="title">
      <string>LODs</string>
     </property>
     <addaction name="actionAll_LODs"/>
     <addaction name="actionLOD0"/>
     <addaction name="actionLOD1"/>
     <addaction name="actionLOD2"/>
     <addaction name="actionLOD3"/>
     <addaction name="actionLOD4"/>
    </widget>
    <addaction name="menuLODs"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuScene"/>
   <addaction name="menuTools"/>
   <addaction name="menuGenerate_2"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDebug"/>
   <addaction name="menuTest"/>
   <addaction name="menuPlugins"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBuilding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Building</string>
   </property>
  </action>
  <action name="actionFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Face</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset Selection</string>
   </property>
  </action>
  <action name="actionShow_temporal_tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show temporal tools</string>
   </property>
  </action>
  <action name="actionLoad_recursive">
   <property name="text">
    <string>Load recursive...</string>
   </property>
  </action>
  <action name="actionDelete_node">
   <property name="text">
    <string>Delete node</string>
   </property>
  </action>
  <action name="actionExport_osg">
   <property name="text">
    <string>Export osg</string>
   </property>
  </action>
  <action name="actionForce_LOD0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD0</string>
   </property>
  </action>
  <action name="actionForce_LOD1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD1</string>
   </property>
  </action>
  <action name="actionForce_LOD2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD2</string>
   </property>
  </action>
  <action name="actionForce_LOD3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD3</string>
   </property>
  </action>
  <action name="actionForce_LOD4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD4</string>
   </property>
  </action>
  <action name="actionExport_JSON">
   <property name="text">
    <string>Export JSON</string>
   </property>
  </action>
  <action name="actionAuto_LOD">
   <property name="text">
    <string>Auto LOD</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionClearSelection">
   <property name="text">
    <string>ClearSelection</string>
   </property>
  </action>
  <action name="actionAll_LODs">
   <property name="text">
    <string>All LODs</string>
   </property>
  </action>
  <action name="actionLOD0">
   <property name="text">
    <string>LOD0</string>
   </property>
  </action>
  <action name="actionLOD1">
   <property name="text">
    <string>LOD1</string>
   </property>
  </action>
  <action name="actionLOD2">
   <property name="text">
    <string>LOD2</string>
   </property>
  </action>
  <action name="actionLOD3">
   <property name="text">
    <string>LOD3</string>
   </property>
  </action>
  <action name="actionLOD4">
   <property name="text">
    <string>LOD4</string>
   </property>
  </action>
  <action name="actionShadows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadows</string>
   </property>
  </action>
  <action name="actionShadows_options">
   <property name="text">
    <string>Shadows options...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionInfo_bubbles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info bubbles</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExport_tiled_osga">
   <property name="text">
    <string>Export tiled osga</string>
   </property>
  </action>
  <action name="actionShow_advanced_tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show advanced tools</string>
   </property>
  </action>
  <action name="actionDump_osg">
   <property name="text">
    <string>Dump osg</string>
   </property>
  </action>
  <action name="actionExport_citygml">
   <property name="text">
    <string>Export citygml</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionLoad_bbox">
   <property name="text">
    <string>Load bbox...</string>
   </property>
  </action>
  <action name="actionCompute_files_bbox">
   <property name="text">
    <string>Compute files bbox</string>
   </property>
  </action>
  <action name="actionAdd_Tag">
   <property name="text">
    <string>Add Tag</string>
   </property>
  </action>
  <action name="actionAdd_Flag">
   <property name="text">
    <string>Add Flag</string>
   </property>
  </action>
  <action name="actionTest_1">
   <property name="text">
    <string>Test 1</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>Test 2</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="text">
    <string>Test 3</string>
   </property>
  </action>
  <action name="actionTest_4">
   <property name="text">
    <string>Test 4</string>
   </property>
  </action>
  <action name="actionTest_5">
   <property name="text">
    <string>Test 5</string>
   </property>
  </action>
  <action name="actionDump_scene">
   <property name="text">
    <string>Dump scene</string>
   </property>
  </action>
  <action name="actionDump_selected_nodes">
   <property name="text">
    <string>Dump selected nodes</string>
   </property>
  </action>
  <action name="actionFix_building">
   <property name="text">
    <string>Fix building</string>
   </property>
  </action>
  <action name="actionOptim_osg">
   <property name="text">
    <string>Optim osg</string>
   </property>
  </action>
  <action name="actionExport_OBJ">
   <property name="text">
    <string>Export OBJ</string>
   </property>
  </action>
  <action name="actionExport_OBJ_split">
   <property name="text">
    <string>Export OBJ split</string>
   </property>
  </action>
  <action name="actionExport_OBJ_Assimp">
   <property name="text">
    <string>Export OBJ (Assimp)</string>
   </property>
  </action>
  <action name="actionChange_Detection">
   <property name="text">
    <string>Change Detection</string>
   </property>
  </action>
  <action name="actionOBJ_to_CityGML">
   <property name="text">
    <string>Convert OBJ to CityGML</string>
   </property>
  </action>
  <action name="actionFff">
   <property name="text">
    <string>fff</string>
   </property>
  </action>
  <action name="actionCut_CityGML_with_Shapefile">
   <property name="text">
    <string>Cut CityGML with Shapefile</string>
   </property>
  </action>
  <action name="actionScene_GenerateLOD0">
   <property name="text">
    <string>LOD0</string>
   </property>
  </action>
  <action name="actionScene_GenerateLOD1">
   <property name="text">
    <string>LOD1</string>
   </property>
  </action>
  <action name="actionShp_Extrusion">
   <property name="text">
    <string>Shp Extrusion</string>
   </property>
  </action>
  <action name="actionVisibilite">
   <property name="text">
    <string>Visibilite...</string>
   </property>
  </action>
  <action name="actionSplit_CityGML_Buildings">
   <property name="text">
    <string>Split CityGML Buildings</string>
   </property>
  </action>
  <action name="actionCut_CityGML_with_Cadastre">
   <property name="text">
    <string>Cut CityGML with Cadastre</string>
   </property>
  </action>
  <action name="actionTiling_CityGML">
   <property name="text">
    <string>Tiling CityGML</string>
   </property>
  </action>
  <action name="actionCreate_Roads_on_MNT">
   <property name="text">
    <string>Create Roads on MNT</string>
   </property>
  </action>
  <action name="actionCreate_Vegetation_on_MNT">
   <property name="text">
    <string>Create Vegetation on MNT</string>
   </property>
  </action>
  <action name="actionCut_MNT_with_Shapefile">
   <property name="text">
    <string>Cut MNT with Shapefile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/vcity.qrc"/>
 </resources>
 <connections/>
</ui>
